Google.Api.Generator/Generation/SnippetCodeGenerator.cs

```diff
@@ -149,10 +153,90 @@ private static IEnumerable<SnippetDef> Snippets(SourceFileContext ctx, ServiceDe
         private SnippetDef Snippet { get; }
         public string SnippetMethodName => Snippet.SnippetMethodName;
 
-        public (CompilationUnitSyntax, SnippetMetadata) Generate(SourceFileContext ctx)
+        public (string, SnippetMetadata) Generate(SourceFileContext ctx)
```

Comment:
I can see why we need to do this - but I don't really like it. It feels like if we change the formatting centrally, we'll need to do it here as well.

I don't have a better solution though.